tasker/task/forms.py

```diff
@@ -3,10 +3,11 @@
 from wtforms.fields.html5 import DateField
 from wtforms.validators import DataRequired
 
-class Task(FlaskForm):
+class TaskForm(FlaskForm):
     name = StringField('Name', validators=[DataRequired()])
     description = StringField('Description', validators=[DataRequired()])
     due_date = DateField('Due date', validators=[DataRequired()])
+    job_template_id = IntegerField('Template', validators=[DataRequired()])
```

Comment:
question in slack - do one off tasks need a job template?

Reply:
I think there should be an hour field too, like a job template so that the task is not due at midnight right as the day starts.